demos/README.md

```diff
@@ -214,6 +241,34 @@ For example:
 cmake -DCMAKE_BUILD_TYPE=Release -DENABLE_PYTHON=ON <open_model_zoo>/demos
 ```
 
+### <a name="build_specific_demos"></a>Build Specific Demos
+
+To build specific demos, follow the instructions for building the demo applications above,
+but add `--target <list of demos>` to the `cmake` or `--target="<list of demos>"` to the `build_demos*` command.
```

Comment:
```suggestion
but add `--target <list of demos>` to the `cmake --build` command or `--target="<list of demos>"` to the `build_demos*` command.
```

I would also suggest replacing `<list of demos>` with `<demo1> <demo2> ...` to make it clear how the list is to be formatted.

Reply:
Done.